samples/dotnet/EdgeX509AuthDownstreamDevice/Program.cs

```diff
@@ -271,32 +299,26 @@ static void InstallChainCertificates(IEnumerable<X509Certificate2> certificateCh
         ///    a prompt will likely come up to confirm the installation of the certificate.
         ///    This usually happens the first time a certificate will be installed.
         /// </summary>
-        static void InstallCACert()
+        static void InstallCACert(X509Certificate2 trustedCACert)
         {
-            string certPath = Environment.GetEnvironmentVariable("IOTEDGE_TRUSTED_CA_CERTIFICATE_PEM_PATH");
-            if (!string.IsNullOrWhiteSpace(certPath))
-            {
-                Console.WriteLine("User configured CA certificate path: {0}", certPath);
-                if (!File.Exists(certPath))
-                {
-                    // cannot proceed further without a proper cert file
-                    Console.WriteLine("Invalid certificate file: {0}", certPath);
-                    throw new InvalidOperationException("Invalid certificate file.");
-                }
-                else
-                {
-                    Console.WriteLine("Attempting to install CA certificate: {0}", certPath);
-                    X509Store store = new X509Store(StoreName.Root, StoreLocation.CurrentUser);
-                    store.Open(OpenFlags.ReadWrite);
-                    store.Add(new X509Certificate2(System.Security.Cryptography.X509Certificates.X509Certificate.CreateFromCertFile(certPath)));
-                    Console.WriteLine("Successfully added certificate: {0}", certPath);
-                    store.Close();
-                }
-            }
-            else
+            Console.WriteLine("User configured CA certificate path: {0}", TrustedCACertPath);
```

Comment:
Shouldn't this be trustedCACert?

Reply:
Here and in the function

Reply:
We add trustedCACert to the store @ line 308. trustedCACert is the certificate object, not the string representation.

Reply:
Yep. Okay makes sense.